src/main/java/net/mcreator/element/types/LivingEntity.java

```diff
@@ -172,6 +173,7 @@
 	public int spawningProbability;
 	public String mobSpawningType;
 	public int minNumberOfMobsPerGroup;
+	public int[] spawnPercentageRaid;

```

Comment:
Better name would be raidSpawnPercentage

Reply:
Actually, what part of this is percentage? It is count of spawns, not percentage so rename this. You should also rename translation keys and help tips